### PR TITLE
guest_agent: update the param name as the meaning have changed

### PR DESCRIPTION
**Pull request by yanan-fu**
The interface of class 'Monitor' have changed, update the param
name to make it more readable

id: 1806375
Signed-off-by: Yanan Fu <yfu@redhat.com>

### CODE REVIEW

**Comment by yanan-fu** [comment]
@xutian could you help review, together with https://github.com/autotest/tp-qemu/pull/2109, thanks!

File: virttest/guest_agent.py [hunk 1]
```diff
@@ -154,7 +154,7 @@ def __init__(self, vm, name, serial_type, serial_filename,
         :param name: Guest agent identifier.
         :param serial_type: Specific which serial type (firtio or isa) guest
                 agent will use.
-        :param serial_filename: Guest agent socket filename.
+        :param gagent_params: Guest agent params.
```
Comment (xiagao) [inline virttest/guest_agent.py]: could you make this param more clear?

Comment (yanan-fu) [reply virttest/guest_agent.py]: @xiagao  It means all the params the object `agent_name`, it is clear enough for me.
What info are you expect ?

Comment (xutian) [reply virttest/guest_agent.py]: I guess xigao need a example: 
guest agent test params object, content like {'monitor_filename': file_path}

Comment (yanan-fu) [reply virttest/guest_agent.py]: @xiagao @xutian updated with more detailed info, please help review again, thanks!

**Comment by xiagao** [comment]
Acked-by: Xiaoling Gao <xiagao@redhat.com>

**Comment by xutian** [comment]
ACK

**Pull request merged**